package.json

```
@@ -18,7 +18,7 @@
     "eject": "react-scripts eject",
     "lint": "eslint \"src/**/*.js\"",
     "start": "react-scripts start",
-    "test": "react-scripts test",
+    "dev": "./node_modules/.bin/react-scripts start",
```

This can simply be written as:
```suggestion
    "dev": "react-scripts start",
```
So, it does the same job the **start** script does.